modules/standard/FileSystem.chpl

```diff
@@ -675,7 +776,7 @@ proc exists(out error: syserr, name: string): bool {
 */
 
 iter findfiles(startdir: string = ".", recursive: bool = false,
-               hidden: bool = false): string {
+               hidden: bool = false): bytes {
```

Comment:
string literal should be b"" in the below

Reply:
For findfiles specifically, if it works, it might make more sense to call a generic version or call the bytes version from the string version. This is OK if the others have some problem.